web-console/src/views/query-view/query-output/query-output.tsx

```diff
@@ -239,12 +239,14 @@ export const QueryOutput = React.memo(function QueryOutput(props: QueryOutputPro
       const sorted = parsedQuery.getSorted();
       if (sorted) {
         className.push(
-          sorted.map(sorted => {
-            if (sorted.id === header) {
-              return sorted.desc ? '-sort-desc' : '-sort-asc';
-            }
-            return '';
-          })[0],
+          sorted
+            .map(sorted => {
+              if (sorted.id === header) {
+                return sorted.desc ? '-sort-desc' : '-sort-asc';
+              }
+              return '';
+            })
+            .join(''),
```

Comment:
This definitely does not make sense this could create classes like `-sort-desc-sort-asc`

The existing code probably does not make sense either as it will add the `undefined` class.